### DISCUSSION TITLE
Handover domain freetz.org

### DISCUSSION THREAD

**Discussion by kriegaex** [Freetz-NG/freetz-ng#1075] [Allgemein] [answered]
Hello.

For many years, I have been sponsoring the domain freetz.org which also redirects to Freetz-NG. Despite having retired from Freetz a long time ago, it was fine for me to continue this small kind of sponsorship for the project. Now, I think the time has come to either cancel the domain or hand it over to whoever is maintaining this project. So what shall it be? I make it up to you. 🙂

Kind regards

Alexander Kriegisch
Freetz co-founder

**Comment by fda77** [comment]
-- original version below
Hello, thank you for taking on the legal risk for the domain and the costs for a long time. First of all, I don't know how you keep track of things here, in Trac+svn I used the name cuma. I don't want to register a domain in my name for this myself, there are too many crazy people in the Freetz environment. The first thing that comes to mind is of course Hippi, who would be happy if something was torn off my bones etc. and has been threatening me with the police for years. This can be found on the page for every Fritzbox, e.g. https://boxmatrix.info/wiki/FRITZ!Box_7690#Modding The reason was: https://freetz-ng.github.io/freetz-ng/NEWS.html#2022-07-03
I'm glad that my address isn't in the whois, I have no idea about anonymizing domains. So I politely decline.
I can't think of anyone else, maybe someone else will get in touch.

--
Hallo, danke dass du lange das rechtliche Risiko für die Domain und die Kosten übernommen hast. Zuerst mal, ich weiss nicht wie wie du hier den Überblick hast, im Trac+svn hatte ich den Namen cuma benutzt. Ich selbst mag keine Domain auf meinen Namen hierfür registrieren, es gibt zu viele Irre im Freetz-Umfeld. Zuerst fällt mir da natürlich Hippi ein, den es freun würde wenn etwas von meinen Knochen abgerissen würde etc und mir seit Jahren mit der Polizei drohe. Das ist auf der Seite zu jeder Fritzbox zu finden, zb https://boxmatrix.info/wiki/FRITZ!Box_7690#Modding Ursache war: https://freetz-ng.github.io/freetz-ng/NEWS.html#2022-07-03
Da bin ich froh wenn meine Adresse nicht im whois steht, von Anonymisierung von Domains hab ich keine Ahnung. Daher lehne ich dankend ab.
Mir fällt auch sonst niemand ein, vielliecht meldet sich sonst jemand.

EDIT: Ich wusst übrigens gar nicht dass es einen Redirect hierher gab, das hätte man auf Github direkt einbauen können
![image](https://github.com/user-attachments/assets/78be3211-20cd-45b3-9c22-113e2db548b5)

**Comment by kriegaex** [comment]
Well, if I cancel the domain, the Freetz community will also lose options to use e-mail addresses (there are a few foo@freetz.org ones), mailing lists and web space. BTW, any new domain owner could receive e-mails to those e-mail addresses.

As for the fear of the legal implications of being a domain owner, I am bearing that risk, even though I have not been involved with Freetz or the contents of its documentation for a long time. Until today, nobody ever tried to attack me there, neither AVM nor any troll.

**Comment by fda77** [comment, marked as answer]
I know! I still have fetchmail running for this account. The last 10 years were without mails :)
But I really don't want to take the legal risk and/or pay 15eu/year (donation system here "doesn't work")
And i dont know anybody who could do it.
By the way, AVM was sold a few months ago

**Reply by kriegaex** [reply]
Fine. I just initiated cancellation of freetz.org.

**Reply by fda77** [reply]
You could ask other people like @harryboo , @er13 or even @meiser79

**Reply by Conan179** [reply]
@fda77 can you write me on, on Discord? (https://discord.com/channels/1315337032775630908/1315337032775630911)

**Reply by fda77** [reply]
@Conan179  "Conan179 nimmt keine Freundschaftsanfragen an"

**Reply by Conan179** [reply]
ah das war aus, jetzt dürfte es gehen

**Reply by kriegaex** [reply]
> You could ask other people like @harryboo , @er13 or even @meiser79

Sorry, too late, the domain is unregistered already. Feel free to re-register it, as long as it is available.

**Comment by fda77** [comment]
Aktuell für mickrige 81,5 dollar zu haben
 -  `https://www.gname.com/auction/freetz_org`
 - 34e6c85ee

**Comment by kriegaex** [comment]
Von mir hättest Du sie kostenlos übertragen bekommen - ohne doofen Unterstrich.

**Comment by fda77** [comment]
Das fand ich sehr nett, danke. Ich bin aber nicht bereit hier auch noch je nach Anbieter 15-25 Euro auszugeben.
Es hätte sich auch sonst jemand melden können, Zeit war genug.
